docs/resources.rst

```diff
@@ -177,3 +177,27 @@ Virtual Switches
 .. autoclass:: zhmcclient.VirtualSwitch
    :members:
    :special-members: __str__
+
+
```

Comment:
The "Metrics" section should not be in the "Resources" chapter of the documentation, because it is not really a resource. Either we create a "Services" chapter, or we add it to the "General features" chapter.